interop/kotlinx-metadata/api/kotlinx-metadata.api

```diff
@@ -1,97 +1,3 @@
-public final class com/squareup/kotlinpoet/metadata/FlagsKt {
```

Comment:
Is it worth it being super diligent and deprecating all of these with `@ReplaceWith`?

Reply:
I don't think so, especially as many of them are just import changes

Reply:
Might make sense to call out the deletions in the release notes.